src/index.js

```diff
@@ -10,7 +11,8 @@ import store from 'store'
 
 const defaultBridgeServer = 'https://bridge.originprotocol.com'
 const defaultIpfsDomain = 'gateway.originprotocol.com'
-const defaultDiscoveryServer = 'https://discovery.originprotocol.com'
+//const defaultDiscoveryServer = 'https://discovery.originprotocol.com'
```

Comment:
Let's follow the same pattern as other configs. Default is what will be used in prod.
Then for local development we use env files:
https://github.com/OriginProtocol/origin-box/blob/master/envfiles/origin-dapp.env
https://github.com/OriginProtocol/origin-dapp/blob/master/dev.env